endpointmanager/pkg/fetcher/fetcher.go

```diff
@@ -53,15 +53,15 @@ func GetEndpointsFromFilepath(filePath string, source string, listURL string) (L
 		return ListOfEndpoints{}, nil
 	}
 
-	validSource := helpers.StringArrayContains(sources, source)
-	if validSource {
-		return GetListOfEndpointsKnownSource([]byte(byteValue), source, listURL)
+	validFormat := helpers.StringArrayContains(formats, source)
```

Comment:
I think we'd want to check the `format` here, not the `source`, which then means we can remove "CareEvolution" and "1Up" as `formats` above